crates/control/src/ball_filter.rs

```diff
@@ -463,18 +327,15 @@ fn project_to_image(
 }
 
 fn is_visible_to_camera(
-    hypothesis: &Hypothesis,
+    ball: &FilteredBall<Ground>,
     camera_matrix: &CameraMatrix,
     ball_radius: f32,
     projected_limbs: &[Limb],
-    configuration: &BallFilterParameters,
 ) -> bool {
-    let position_on_ground = hypothesis.selected_ball_position(configuration).position;
-    let position_in_image =
-        match camera_matrix.ground_with_z_to_pixel(position_on_ground, ball_radius) {
-            Ok(position_in_image) => position_in_image,
-            Err(_) => return false,
-        };
+    let position_in_image = match camera_matrix.ground_with_z_to_pixel(ball.position, ball_radius) {
+        Ok(position_in_image) => position_in_image,
+        Err(_) => return false,
+    };
     (0.0..640.0).contains(&position_in_image.x())
         && (0.0..480.0).contains(&position_in_image.y())
```

Comment:
try to remove these magic values...

Reply:
Voting for doing this in a separate PR for all camera magic values since this requires exposing hardware parameters which can't we do yet as far as I know.

Reply:
Then we should better open an issue for that. Is there something like that already out there?

Reply:
yes, #751